pingdom/src/main/java/com/pulse/service/PingerService.java

```diff
@@ -0,0 +1,46 @@
+package com.pulse.service;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+public class PingerService implements Runnable {
+    public String pingUrl;
+    public PingerService(){
+
+    }
+
+    public PingerService(String pingUrl) {
+        System.out.println("assigned");
```

Comment:
why do we need logs in a constructor? 🤔

Reply:
Have to remove that log. That is for my debugging purpose. I'll remove that log